sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/TypeHelper.cs

```diff
@@ -67,6 +67,25 @@ public static bool Is(this ITypeSymbol typeSymbol, KnownType type)
             return typeSymbol != null && IsMatch(typeSymbol, type);
         }
 
+        public static bool IsAny(this ITypeSymbol typeSymbol, params KnownType[] types)
+        {
+            if (typeSymbol == null)
+            {
+                return false;
+            }
+
+            // For is twice as fast as foreach on ImmutableArray so don't use Linq here

```

Comment:
_types_ is a simple array not an _ImmutableArray_. Is this comment still relevant?